base_domain_operator/expression.py

```diff
@@ -22,12 +22,13 @@ def __init__(self, leaf, model, join_context=None, internal=False):
 
 
 class Expression(expression):
-    def parse(self, cr, uid, context):  # pylint: disable=R8110
+
+    def parse(self):
         domain = []
-        env = api.Environment(cr, uid, context)
+        env = api.Environment(*self.root_model.env.args)
```

Comment:
this will end up to be the same env as `root_model`'s anyways: https://github.com/OCA/OCB/blob/10.0/odoo/api.py#L735 - so you can just as well assign it